results-tabulation-api/ext/ExtendedElection/ExtendedElectionProvincialCouncilElection2021/ExtendedFunctions/get_area_map_query.py

```diff
@@ -0,0 +1,110 @@
+from app import db
+from sqlalchemy import bindparam
+from sqlalchemy.orm import aliased
+from constants.VOTE_TYPES import NonPostal
+
+
+def get_area_map_query(election):
+    from orm.entities import Election, Area
+    from orm.entities.Area import AreaAreaModel
+    from orm.enums import AreaTypeEnum
+
+    country = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.Country).subquery()
+    administrative_district = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.AdministrativeDistrict).subquery()
+    province = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.Province).subquery()
+    polling_division = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.PollingDivision).subquery()
+    polling_district = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.PollingDistrict).subquery()
+    polling_station = db.session.query(Area.Model).filter(
+        Area.Model.areaType == AreaTypeEnum.PollingStation).subquery()
+    counting_centre = db.session.query(Area.Model).filter(
```

Comment:
I think polling division is also should be here

Reply:
polling division is already there.